src/Oryx.Common/Common.csproj

```diff
@@ -13,4 +13,11 @@
   </PropertyGroup>
 
   <Import Project="$(MSBuildThisFileDirectory)\..\CommonFiles\AssemblyVersion.proj" />
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.5" />
+    <PackageReference Include="Nett" Version="0.13.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="YamlDotNet" Version="8.0.0" />
+  </ItemGroup>
```

Comment:
Are these packages used by common proj

Reply:
I put these in Common because we still have Hugo related stuff in BuildScriptGenerator. I need them in common so that newly moved Hugo related stuff in Detector library can use these too.